appengine/firebase-tictactoe/README.md

```diff
@@ -0,0 +1,62 @@
+# Tic Tac Toe on Google App Engine Standard using Firebase
+
+This directory contains a project that implements a realtime two-player game of
+Tic Tac Toe on Google [App Engine Standard][standard], using the [Firebase] database
+for realtime notifications when the board changes.
+
+[Firebase]: https://firebase.google.com
+[standard]: https://cloud.google.com/appengine/docs/about-the-standard-environment
+
+## Prerequisites
+
+* Install [Apache Maven][maven] 3.0.5 or later
+* Create a project in the [Firebase Console][fb-console]
+* Install the [Google Cloud SDK][sdk]
+* For staging locally, you must supply credentials that would otherwise be
```

Comment:
"staging locally" -> "local development".  `gcloud auth application-default login` is preferred to `GOOGLE_APPLICATION_CREDENTIALS`